document virtualColumns in native query documentation, fix some redirects (#12917)
apache/druid: docs/querying/topnquery.md

|intervals|A JSON Object representing ISO-8601 Intervals. This defines the time ranges to run the query over.|yes|
|granularity|Defines the granularity to bucket query results. See [Granularities](../querying/granularities.md)|yes|
|filter|See [Filters](../querying/filters.md)|no|

> |virtualColumns|Define any [virtual columns](./virtual-columns.md) to be available to use as the grouping `dimension` or referenced as inputs to `aggregations` or `postAggregations`.| no (default none)|

Reviewer: ```suggestion
|virtualColumns|A JSON list of [virtual columns](./virtual-columns.md). You can reference a virtual column as the grouping `dimension` or as an input in `aggregations` or `postAggregations`.| no (default none)|
```
